### PR TITLE
Adding detekt support

### PR DESCRIPTION
**Pull request by gmerinojimenez**
To be included in a github action

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    
+    - name: "Detekt"
+      uses: vgaidarji/android-github-actions-build@v1.0.1
+      with:
+        args: 'bash ./gradlew --no-daemon detekt'
```
Comment (dpastor) [inline .github/workflows/tests.yml]: and "check" ?

Comment (gmerinojimenez) [reply .github/workflows/tests.yml]: sure, added too

File: app/build.gradle [hunk 2]
```diff
@@ -35,6 +36,6 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.3.2'
     implementation 'androidx.appcompat:appcompat:1.2.0'
```
Comment (dpastor) [inline app/build.gradle:38]: $support_version also here?

**Review by dpastor — APPROVED** [review]

File: app/build.gradle (hunk 2 above)
Comment (gmerinojimenez) [reply app/build.gradle:38]: 1.2.1 is not supported in appcompat:
https://developer.android.com/jetpack/androidx/releases/appcompat

**Pull request merged**